hw3/01-characters.js

```diff
@@ -1,4 +1,44 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-fetch(url);
+const parentDiv = document.createElement('div');
+parentDiv.classList.add('parent-div-01');
+
+const section = document.getElementById('section-01');
+section.appendChild(parentDiv);
+
+function createCharacterCard(character) {
```

Comment:
According to the Airbnb style guide [7.1](https://github.com/airbnb/javascript#functions--declarations) named function expressions should be used instead of function declarations.

Reply:
Addressed.